packaging/smp/config/db/db.changelog.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog  http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.18.xsd">
+
+  <changeSet author="harmony" id="2.0.0-migration">
+    <preConditions onFail="MARK_RAN" onError="HALT">
+      <changeSetExecuted id="0002:001" author="root" changeLogFile="/opt/harmony-smp/db/migrations/0002-Upgrade_4.2.sql"/>
+    </preConditions>
+    <sqlFile dbms="mysql" relativeToChangelogFile="true" path="migrations/mysql5innoDb-4.2_to_5.0-step_01.sql" />
+    <sqlFile dbms="mysql" relativeToChangelogFile="true" path="migrations/mysql5innoDb-4.2_to_5.0-step_02.sql" />
+    <rollback>
+      <sqlFile dbms="mysql" relativeToChangelogFile="true" path="migrations/mysql5innoDb-4.2_to_5.0-rollback.sql" />
+    </rollback>
+  </changeSet>
+
+  <changeSet author="harmony" id="2.0.0-migrate-properties">
+    <preConditions onFail="MARK_RAN" onError="HALT">
+      <changeSetExecuted id="0002:001" author="root" changeLogFile="/opt/harmony-smp/db/migrations/0002-Upgrade_4.2.sql"/>
+      <sqlCheck  expectedResult="1">SELECT COUNT(*) FROM SMP_CONFIGURATION WHERE PROPERTY_NAME='configuration.dir'</sqlCheck>
+    </preConditions>
+    <sql dbms="mysql"><![CDATA[
+      INSERT IGNORE INTO SMP_CONFIGURATION (PROPERTY_NAME, CREATED_ON, LAST_UPDATED_ON, PROPERTY_VALUE) VALUES 
```

Comment:
Why "INSERT IGNORE" - will it also work if for example only one of these rows will be inserted?

Reply:
"insert ignore" ignores failures caused by possible primary key violations, so yes it should.